cmd/snap/cmd_model.go

```diff
@@ -123,21 +150,86 @@ func (x *cmdModel) Execute(args []string) error {
 		termWidth = 100
 	}
 
+	esc := x.getEscapes()
+
 	w := tabWriter()
+	defer w.Flush()
```

Comment:
there are several `return <error>` where we *don't* want to print what we've accumulated into the tabwriter (like the StoreAccount retrieval error, or the several invalidTypeErrors)